mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -36,18 +36,27 @@ function validate50PercentButton (loanTokenSymbol: string, paymentTokenSymbol: s
   if (hasSufficientBalance) {
     cy.getByTestID('payback_input_text_error').should('not.exist')
     cy.getByTestID('button_confirm_payback_loan_continue').should('not.have.attr', 'aria-disabled')
+    cy.getByTestID('loan_outstanding_balance').invoke('text').then(text => {
+      const outstandingBalance = new BigNumber(text.replace(loanTokenSymbol, '').trim())
+      if (loanTokenSymbol === paymentTokenSymbol) {
+        cy.getByTestID('payback_input_text').should('have.value', outstandingBalance.div(2).toFixed(8))
```

Comment:
can we add a test when `loanTokenSymbol !== paymentTokenSymbol`?

Reply:
Sure, updated readability and more tests for 50% and 100% quick select buttons